libglusterfs/src/common-utils.c

```diff
@@ -3682,22 +3699,38 @@ gf_is_same_address(char *name1, char *name2)
     gf_boolean_t ret = _gf_false;
     int gai_err = 0;
     struct addrinfo hints;
+    xlator_t *this = THIS;
+    glusterfs_ctx_t *ctx = this->ctx;
+    gf_boolean_t flag = _gf_false;
+
+    if (ctx && ctx->dnscache)
+        flag = _gf_true;
```

Comment:
It would be better to use a more meaningful name, like "have_cache" or something like that.

Reply:
done